packages/@react-types/list/package.json

```diff
@@ -15,6 +15,9 @@
   "peerDependencies": {
     "react": "^16.8.0 || ^17.0.0-rc.1 || ^18.0.0"
   },
+  "devDependencies": {
+    "@react-spectrum/dnd": "3.0.0-alpha.6"
+  },
```

Comment:
I think this is ok? Moving the definition of DnDHooks into the types was a bit troublesome from what I remember (my notes from when I tried this don't mention why, need to check again)

Reply:
Ah I remember why, it would involve moving the dnd options/return types into `@react-types` -> requires moving things like `MultipleSelectionManager`'s types + `DragPreview` type. Lemme know if we wanna port all this stuff into `@react-types`

Reply:
i assume this devDep only exists for the stories? or is it for the types?
listview doesn't depend on the dnd implementation packages, except for types? if so, then those should be in the react-types package I think

Reply:
This devDep is for https://github.com/adobe/react-spectrum/blob/dafd28698cea084d38a039a8557a3bcacf101cdb/packages/%40react-types/list/src/index.d.ts#L24
which is for https://github.com/adobe/react-spectrum/blob/dafd28698cea084d38a039a8557a3bcacf101cdb/packages/%40react-types/list/src/index.d.ts#L65
so it is for the types (aka listview only depends on the dnd implementation types as you mentioned). I figured since it is just for the `DnDHooks` type that we could get away with a devDep instead since we would otherwise have to move a bunch of other stuff into react-types instead (MultipleSelectionManager's types + DragPreview type)